### DISCUSSION TITLE
Wrong time zone after Daylight Saving Time change on to-dos

### DISCUSSION THREAD

**Discussion by sysmin1** [Leantime/leantime#2767] [Q&A] [answered]
Recently, my country changed local time ([GMT+1, one hour back](https://greenwichmeantime.com/time-zone/rules/eu/)). Still, after double checking Leantime, comments and updates show as one hour forward from the current date. I'd love to hear from the team, not sure if it's a bug proper or misconfiguration on my side. Big thanks to the team for this project.

**Steps to reproduce:**
1. I edited this variable in **/etc/php/8.2/cli/php.ini**, which was unused since install.
> date.timezone = Europe/Madrid
2. And edited this variable too, which seemed to work until now, at **/var/www/leantime/config/configuration.php**.
> public $defaultTimezone = 'Europe/Madrid';

**Expected behavior**
Correct time on comments and website notifications.

**What set up are you using**
Installed as a container from [ISO 18.1 at TurnKey Linux](https://www.turnkeylinux.org/leantime) on Proxmox Virtual Enviroment. Debian base, container shows correct time.
**Leantime Version**
Leantime 3.1.2.
**Server**
NGINX
**PHP / MySQL Version**
myslq 8.2

**Additional context**
- [Same issue as this user](https://github.com/Leantime/leantime/issues/2455), but I didn't change the database file.
- On a separate note, email notifications always return a login screen, even after login in. Editing the second file's variable like this doesn't seem to fix it. I'm having second thoughts if it's related to time management too,

> public $sessionExpiration = 604800;

**Comment by marcelfolaron** [comment]
Hey,
The timezone can also be set by the user individually under your profile settings. This may be why the timezone shift happened. 
The login issue should be resovled in 3.3.0

**Comment by sysmin1** [comment]
It seems updating fixed the email issue. Timezone may be ok for now, comments and tasks work, only the notification bell on the upper right corner was off.

I'm getting error 403 way too often while clicking through projects, d'oh. Last time, editing my php.ini file and reloading Apache for these variables gave the installation more wiggle room to avoid crashes: max_input_vars, memory_limit, post_max_size.

**Reply by sysmin1** [reply, marked as answer]
Update: I fixed the 403 errors too. Apache modules had to be disabled or edited for higher values at /etc/apache2/mods-available:
- a2dismod security2
- a2dismod evasive
- mpm_prefork.conf
- reqtimeout.conf

Remember to restart Apache too (systemctl restart apache2), and thanks again for this project.

**Comment by gergely-szocs-hf** [comment]
The OS timezone has been set to CET, the APP timezone as well and when I check the timezone settings in the user settings it is also CET however I still have the time mismatch issue.

<img width="1959" height="1069" alt="image" src="https://github.com/user-attachments/assets/ecd6e4b9-db87-47b8-9f35-d059a10c9559" />
